draft-ietf-rats-architecture.md

```diff
@@ -529,76 +529,95 @@ This document uses the following terms.
 Attester:
 
 : A role performed by an entity (typically a device) whose Evidence must be appraised in order to infer the extent to which the Attester is considered trustworthy, such as when deciding whether it is authorized to perform some operation.
-Produces: Evidence.
+
+: Produces: Evidence
 
 Relying Party:
 
-: A role performed by an entity that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}.
-Consumes: Attestation Results.
+: A role performed by an entity (typically a device) that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}.
```

Comment:
```suggestion
: A role performed by an entity that depends on the validity of information about an Attester, for purposes of reliably applying application specific actions.  Compare /relying party/ in {{RFC4949}}.
```

Disagree that it's "typically a device".  What it typically is varies by use case.  Such a statement is fine on a per use case basis, but not in the definition of Relying Party in my opinion.